Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -579,7 +578,52 @@ public extension UIView {
     func ancestorView<T: UIView>(withClass name: T.Type) -> T? {
         return ancestorView(where: { $0 is T }) as? T
     }
+}
+
+// MARK: - findConstraint and friends
+public extension UIView {
+    /// SwifterSwift: Search constraints until we find one for the given view and attribute. This will enumerate ancestors since constraints are always added to the common ancestor.
+    ///
+    /// - Parameter view: the view to find
+    /// - Parameter attribute: the attribute to find
+    /// - Returns: matching constraint
+    func findConstraint(view: UIView, attribute: NSLayoutConstraint.Attribute) -> NSLayoutConstraint? {
```

Comment:
What about changing this signature to `findConstraint(attribute:  NSLayoutConstraint.Attribute, at: UIView)` ?

Reply:
`at` feels a little funny to me because it sort of implies that you are only searching "at" that view, when you are actually searching the entire ancestor chain. `for` or `forView` would probably be my preference. I don't have a strong opinion, though. I'll change it.

Reply:
Nice, 'for' would be great :)

Reply:
As the last thing could you just this? Your suggestion is indeed better